exchange/exchange.go

```diff
@@ -245,6 +245,9 @@ func (e *exchange) HoldAuction(ctx context.Context, r *AuctionRequest, debugLog
 		return nil, err
 	}
 
+	// seatNonBid will store nonbids collected inside HoldAuction function, it will not contain the nonbids from stageOutcomes
+	seatNonBid := &openrtb_ext.NonBidsWrapper{}
```

Comment:
we can create a normal object, no need to create a pointer object.

Reply:
done